shopify-client/src/components/Header.vue

```diff
@@ -35,9 +39,15 @@
 <script>
 export default {
   name: "Header",
+  methods:{
+    submitSearch: function(){
+      if(this.keyword){
+        window.location.href = '/app/shops/List?keyword=' + this.keyword;
+      }
+    }
+  },
   computed: {
     cartItems: function() {
```

Comment:
Where is this used?

Reply:
This was supposed to be for the cart tab in the header to show how many items there are in the cart:
https://github.com/mpfan/Mini-Shopify/pull/64#discussion_r597322807

but it was then removed mistakenly from #68, I'm guessing due to a merge conflict.